app/models/user.rb

```diff
@@ -5,6 +5,6 @@ class User < ApplicationRecord
   validates :email, uniqueness: true, presence: true
   validates_presence_of :password
   validates_presence_of :first_name
-  enum role: [:default, :admin]
+  enum role: %i[default admin]
```

Comment:
Brain said these ones he doesn't care about, so we can go into rubocop and figure out how to turn this warning off.